### PR TITLE
Render/reviews

### PR DESCRIPTION
**Pull request by slatzz**

### CODE REVIEW

File: server/db/routes/reviewRoutes.js [hunk 1]
```diff
@@ -1,27 +1,55 @@
...
+            where: {
+              id: guestId,
+            }
+          },
+        })
+        return result;
       })
-      .catch(err => console.warn('Error loading reviews...'))
+      .catch(err => err.message);
+    console.info(reviews);
```
Comment (jstrzesz) [inline server/db/routes/reviewRoutes.js]: remove

File: src/components/landing/auth/Signup.tsx [hunk 2]
```diff
@@ -137,28 +138,30 @@ const SignUp: React.FC<AuthProps> = ({ handleLogin: [isAuth, setAuth] }) => {
...
+              q6={q6}
+              q7={q7}
+              q8={q8}
+              q9={q9}
+              q10={q10}
+              profilePhotoUrl={profilePhotoUrl}
+              setProfilePhotoUrl={setProfilePhotoUrl}
+              onSubmitForm={onSubmitForm}
+            />
+            <br />
```
Comment (jstrzesz) [inline src/components/landing/auth/Signup.tsx]: remove br tag, use css instead

File: src/components/landing/auth/UserDetailsForm.tsx [hunk 3]
```diff
@@ -77,107 +66,109 @@ const UserDetailsForm: React.FC<MyProps> = (Props: MyProps) => {
...
+                    edge="end"
+                  >
+                    {showPassword ? <Visibility /> : <VisibilityOff />}
+                  </IconButton>
+                </InputAdornment>
+              )}
+              labelWidth={70}
+            />
+          </FormControl>
+          <br />
```
Comment (jstrzesz) [inline src/components/landing/auth/UserDetailsForm.tsx]: see above comment

File: src/components/listing/HostReviews.tsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                      fontWeight="fontWeightBold"
+                      borderColor="transparent"
+                    >
+                      {review.user.first_name}
+                      // TODO: Revisiting this
+                      {/* <br />
+                    <Typography variant="caption">
+                      {moment(review.createdAt, 'YYYYMMDD').fromNow()}
+                    </Typography> */}
+                      <br />
```
Comment (jstrzesz) [inline src/components/listing/HostReviews.tsx]: see above comment

File: src/components/listing/UserReviews.tsx [hunk 5]
```diff
@@ -99,10 +100,12 @@ const UserReviews: FC<AppInterface> = ({ user }): JSX.Element => {
             </Tabs>
           </AppBar>
           <TabPanel value={value} index={0}>
-            Reviews left by users about User as a Host.
+            <HostReviews listingId={listingId} allReviews={allReviews} reviewer={reviewer} />
+            <br />
```
Comment (jstrzesz) [inline src/components/listing/UserReviews.tsx]: see above comment

File: src/components/listing/UserReviews.tsx [hunk 6]
```diff
@@ -99,10 +100,12 @@ const UserReviews: FC<AppInterface> = ({ user }): JSX.Element => {
...
           </TabPanel>
           <TabPanel value={value} index={1}>
-            {allReviews}
+            <GuestReviews listingId={listingId} allReviews={allReviews} reviewer={reviewer} />
+            <br />
```
Comment (jstrzesz) [inline src/components/listing/UserReviews.tsx]: see above comment

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

**Review by mconner89 — APPROVED** [review]

**Review by jessrichmond — APPROVED** [review]

**Pull request merged**